src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -242,6 +242,12 @@ public partial class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.Reso
         [PSArgumentCompleter("X64", "Arm64")]
         public string Architecture { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Set this flag to true to get a boost on the performance target of the disk deployed, see here on the respective performance target. This flag can only be set on disk creation time and cannot be disabled after enabled.")]
+        public bool? PerformancePlus { get; set; }
```

Comment:
We'd like suggest to use switch parameter for PerformancePlus in New-Az* command

Reply:
yes, switchParameter would make more sense but all the boolean properties for Disk in this cmdlet [New-AzDiskConfig](https://learn.microsoft.com/en-us/powershell/module/az.compute/new-azdiskconfig?view=azps-9.4.0) are defined as boolean and not Switch. So I wanted to keep it consistent within this cmdlet. 
I think it's better than having one boolean property be switch while ~3 others are all boolean for this cmdlet.

Reply:
@BethanyZhou